src/Nfts/NftsNavigator.tsx

```diff
@@ -0,0 +1,50 @@
+import {createStackNavigator} from '@react-navigation/stack'
+import React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+
+import {Icon} from '../components'
+import {NftRoutes} from '../navigation'
+import {Nfts} from './Nfts'
+
+const Stack = createStackNavigator<NftRoutes>()
+
+export const NftsNavigator = () => {
+  const strings = useStrings()
+
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        headerTitleContainerStyle: {width: '100%', alignItems: 'center'},
+        cardStyle: {backgroundColor: '#fff'},
+      }}
+      initialRouteName="nfts"
+    >
+      <Stack.Screen
+        name="nfts"
+        component={Nfts}
+        options={{
+          title: strings.title,
+          headerLeft: () => null,
```

Comment:
this will hide the back button but won't prevent swiping left to fire `.goBack()`